picard/ui/options/plugins.py

```diff
@@ -306,7 +306,7 @@ def change_details(self):
         self.ui.details.setText("<p>%s</p>" % "<br/>\n".join(text))
 
     def open_plugins(self):
-        files = QtWidgets.QFileDialog.getOpenFileNames(
+        files, _ = QtWidgets.QFileDialog.getOpenFileNames(
```

Comment:
Not so sure about the use of "_" for discarding because of its use as a builtin for gettext.

But other than that LGTM